tesseract_common/include/tesseract_common/yaml_utils.h

```diff
@@ -332,7 +332,8 @@ struct convert<Eigen::VectorXd>
   static Node encode(const Eigen::VectorXd& rhs)
   {
     Node node;
-    for (long i = 0; i < rhs.size(); ++i)
+    long l = static_cast<long>(rhs.size());
+    for (long i = 0; i < l; ++i)
```

Comment:
Can this not be one line `for (long i = 0; i < static_cast<long>(rhs.size()); ++i)` ?